includes/indices/class-algolia-posts-index.php

```diff
@@ -430,4 +430,17 @@ protected function extract_item(Algolia_Task $task) {
 			
 		return get_post( $data['post_id'] );
 	}
+
+	/**
+	 * @param Algolia_Task $task
+	 */
+	public function delete_item( Algolia_Task $task ) {
+		if ( ! isset( $data['post_id'] ) || ! is_int( $data['post_id'] ) ) {
```

Comment:
Wow good catch, didn't push latest code apparently... I have to get some sleep ;) let me fix that.